### PR TITLE
Make multiple borders with box shadow

### PR DESCRIPTION
**Pull request by MhamedEl-shahawy**
I've explained how to create multiple borders with box-shadow this not work exactly like borders, I've explained this has some problems but you can fix this and you can make multiple borders with outline style but outline makes one outer border, not like box-shadow you can make more than one border, I  hope my explanation be more clarity.

### CODE REVIEW

**Comment by AllThingsSmitty** [comment]
This is an interesting tip. I might want to skinny it down a bit, make it less wordy.  Any chance you can trim it down to about half the size?

**Comment by MhamedEl-shahawy** [comment]
> This is an interesting tip. I might want to skinny it down a bit, make it less wordy. Any chance you can trim it down to about half the size?

explanation contains advices and notes if i skinny it down, I think it will miss a bit of value

File: README.md [hunk 1]
```diff
@@ -608,6 +608,40 @@ br + br {
 
 <sup>[back to table of contents](#table-of-contents)</sup>
 
+### Multiple Borders 
+You can use box-shadow to make a border effect, by making the the shadow offset and have 0 blur. Plus, by comma-separating values, you can have as many "borders" as you like:
```
Comment (BenRichter) [inline README.md]:
```suggestion
Use multiple comma-separated box-shadows without offset or blur to add as many "borders" as you like:
```
Shortened. Removed doubled "the"

File: README.md [hunk 2]
```diff
@@ -608,6 +608,40 @@ br + br {
...
+div{
+   background-color:yellowgreen;
+   box-shadow:0 0 0 10px #656;
+}
+```
+This is not particularly impressive, as you can create the same kind of border
+by using the border property. However, the good thing about boxshadow is that we can have as many of them as we want, comma
+separated. So, we can pretty easily add a second deeppink “border”
+to the previous example:
+
```
Comment (BenRichter) [inline README.md]: Redundant. Could be removed entirely.

File: README.md [hunk 3]
```diff
@@ -608,6 +608,40 @@ br + br {
...
+```css
+div{
+   background-color:yellowgreen;
+   box-shadow: 0 0 0 10px #655, 0 0 0 15px deeppink;
```
Comment (BenRichter) [inline README.md]: Maybe easier to understand if you stay with color names for consistency? And unified spacing
```suggestion
   background-color: yellowgreen;
   box-shadow: 0 0 0 10px blue, 
               0 0 0 15px pink;
```

File: README.md [hunk 4]
```diff
@@ -608,6 +608,40 @@ br + br {
...
+}
+```
+#### The shadow solution works quite well in most cases, but has a few caveats :
+- Shadows don’t work exactly like borders, as they don’t affect layout and are
+oblivious to the box-sizing property. However, you can emulate the extra
+space a border would occupy via padding or margins (depending on whether the shadow is inset or not).
```
Comment (BenRichter) [inline README.md]:
```suggestion
#### Caveats :
- Shadows are oblivious to the box-sizing and therefore don’t affect layout. We can add the extra
space a border would occupy via margin or padding (for inset shadows).
```

File: README.md [hunk 5]
```diff
@@ -608,6 +608,40 @@ br + br {
...
+- The method we demonstrated creates fake “borders” on the outside of
+elements. These do not capture mouse events such as hovering or clicking.
+If this is important, you can add the inset keyword to make the shadows
+be drawn on the inside of your element. Note that you will need to add
+extra padding to produce sufficient spacing.
```
Comment (BenRichter) [inline README.md]:
```suggestion
- These fake “borders” are outside of elements and won't capture mouse events such as hovering or clicking.
If required, use the **inset** keyword to make them appear inside. Add padding for sufficient spacing.
```

**Pull request closed without merging**